google-beta/resource_dataflow_job.go

```diff
@@ -580,7 +580,7 @@ func resourceDataflowJobIterateMapHasChange(mapKey string, d *schema.ResourceDat
 	return false
 }
 
-func resourceDataflowJobIsVirtualUpdate(d *schema.ResourceData) bool {
+func resourceDataflowJobIsVirtualUpdate(d *schema.ResourceData, resourceSchema map[string]*schema.Schema) bool {
```

Comment:
How is this supposed to work when `resourceSchema` is never used? I'm unclear on why this method is being changed

Reply:
Good catch! I forgot to remove the `resourceSchema := resourceDataflowJob().Schema` line. This change was supposed to take the resource schema to use as a parameter so that it can be used for both job and flex job resource schemas. I will push a fix for this issue.

Reply:
Alright, I tested and pushed the change. I simply deleted the line that was shadowing the `resourceSchema` parameter.